internal/auth0/client/resource.go

```diff
@@ -1036,6 +1036,22 @@ func NewResource() *schema.Resource {
 								},
 							},
 						},
+						"springcm": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							Computed:    true,
+							MaxItems:    1,
+							Description: "SpringCM SSO configuration.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"acs_url": {
+										Description: "SpringCM ACS URL, for example `https://na11.springcm.com/atlas/sso/SSOEndpoint.ashx`.",
+										Type:        schema.TypeString,
+										Optional:    true,
```

Comment:
No URL validation needed?

Reply:
None present in api2 json schema. The schema here is simply a reflection of that. If any api validaton is happening it will be done within the endpoint itself.